### PR TITLE
Patch/aws rules fix

### PR DESCRIPTION
**Pull request by abhstabs**

### CODE REVIEW

**Comment by pandurangpatil** [comment]
@HiteshMah-Jan can you go through this pull request and approve?

File: rules/sinks/third_parties/sdk/amazon/java.yaml [hunk 1]
```diff
@@ -9,13 +9,13 @@ sinks:
     domains:
       - "amazon.com"
     patterns:
-      - "(?i)(com[.]amazon[.]alexa|com[.]amazon[.]android|com[.]amazon[.]redshift|software[.]amazon[.]awscdk|software[.]amazon[.]awsconstructs|software[.]amazon[.]awssdk|software[.]amazon[.]ion|software[.]amazon[.]jsii|software[.]amazon[.]kinesis|software[.]amazon[.]konstruk|software[.]amazon[.]smithy).*"
```
Comment (HiteshMah-Jan) [inline rules/sinks/third_parties/sdk/amazon/java.yaml:12]: `software[.]amazon[.]awscdk|software[.]amazon[.]awsconstructs` should these also be removed ??

**Review by HiteshMah-Jan — APPROVED** [review]

File: rules/sinks/third_parties/sdk/amazon/java.yaml (hunk 1 above)
Comment (abhstabs) [reply rules/sinks/third_parties/sdk/amazon/java.yaml:12]: Updated the rules.

**Review by pandurangpatil — APPROVED** [review]
looks good

**Pull request merged**